### ISSUE TITLE
CherryPy no longer installable

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-ec2#27]
```
Bootstraping salt master
DEBUG: Running command sudo -S bash -c 'curl -sS -L https://bootstrap.saltstack.com | sh -s -- -d -X -M -N stable' on '52.87.254.124'
DEBUG: Running command sudo -S bash -c 'curl -L https://bootstrap.saltstack.com | sh -s -- -d -X -M -N -P -L -p salt-api stable' on '52.87.254.124'
DEBUG: Uploading file /home/mrocklin/Software/anaconda/lib/python3.4/site-packages/dask_ec2-0.3.0+5.g54cb5fd-py3.4.egg/dask_ec2/templates/auto_accept.conf to /tmp/.__tmp_copy
DEBUG: Running command sudo -S bash -c 'cp -rf /tmp/.__tmp_copy /etc/salt/master.d/auto_accept.conf' on '52.87.254.124'
DEBUG: Running command sudo -S bash -c 'rm -rf /tmp/.__tmp_copy' on '52.87.254.124'
DEBUG: Running command sudo -S bash -c 'pip install cherrypy' on '52.87.254.124'
DEBUG: Attempt 1/3 of function '__install_salt_rest_api' failed
DEBUG: Running command sudo -S bash -c 'pip install cherrypy' on '52.87.254.124'
DEBUG: Attempt 2/3 of function '__install_salt_rest_api' failed
DEBUG: Running command sudo -S bash -c 'pip install cherrypy' on '52.87.254.124'
DEBUG: Attempt 3/3 of function '__install_salt_rest_api' failed
ERROR: 'str' object has no attribute 'decode'
Couldn't install CherryPy. Error is above (maybe try again)
```

I've tried again.  Things are still unhappy.  Maybe a change upstream?

**Comment by mrocklin** [comment]
Fails on both Python 2 and 3

**Comment by danielfrg** [comment]
Fixed on #29
